tests/compile_upp.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Why is this script in `tests` directory?  It should be at the top level.

Reply:
@aerorahul  I use the similar directory structure from the ufs-weather-model. In the future, we would expend more features such as regression test scripts in it.

Reply:
the ufs `compile.sh` calls top-level `build.sh`